templates/worker/01-worker-kubelet/_base/files/kubenswrapper.yaml

```diff
@@ -0,0 +1,12 @@
+# Note: This compatibility wrapper is needed to bridge the gap from OCP 4.11->4.12 when the new 'kubensenter' script is being introduced.
+# It can be removed (and the kubelet.service should call kubensenter directly) when 4.11 is no longer a release we must upgrade from.
+mode: 0755
+path: "/usr/local/bin/kubenswrapper"
```

Comment:
why /usr/local instead of /usr?

Reply:
Ignition can't write to `/usr`.  See https://github.com/openshift/machine-config-operator/issues/3137